app/src/main/java/org/wikipedia/search/SearchResultsFragment.java

```diff
@@ -218,7 +232,7 @@ private void doTitlePrefixSearch(final String searchTerm, boolean force) {
                     searchErrorView.setVisibility(GONE);
                     handleResults(results, searchTerm, startTime);
                 }, caught -> {
-                    searchEmptyView.setVisibility(GONE);
+                    searchErrorView.setVisibility(View.VISIBLE);
```

Comment:
This line is now identical to the next line. What was the rationale of changing it?

Reply:
Done. I guess something went wrong when merging the `master` branch.